app/src/main/resources/organizations.yaml

```diff
@@ -0,0 +1,8 @@
+- name: report-stream
+  description: Report Stream
+  senders:
+    - name: default
+      topic: etor
+      customerStatus: active
+      format: FHIR
+  receivers:
```

Comment:
Will we have a concept of receivers that are tied to a sender?  Maybe in the future?  I'm out of the loop behind what was the thinking in this PR, so... ?

Reply:
Thanks for reviewing. I'm not sure if we'll need receivers. We should discuss it with @jorg3lopez. We're mimicking the structure of the organizations.yaml file used by Report Stream, so this is open to discussion

Reply:
Sure, we can discuss.  I think my overall thought is we should make the setting file our own.  If there's a concept that we don't need for our application, let's not have it.  We can always add concepts back if we ever actually support something.